person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = 0
```

Comment:
- [ ] Please consider using random id. This is because different persons created should have special random id to ensure that the person is unique even if they have similar name or age ☺️.